media/js/base/global.js

```diff
@@ -87,15 +87,16 @@ function getFirefoxMasterVersion(userAgent) {
     return version;
 }
 
+// Used on the plugincheck page to also support all browsers based on Gecko.
+function isLikeFirefox(userAgent) {
+    var ua = userAgent || navigator.userAgent;
+    return (/Iceweasel/i).test(ua) || (/IceCat/i).test(ua) ||
+        (/SeaMonkey/i).test(ua) || (/Camino/i).test(ua);
```

Comment:
Oh, I thought we were replacing like Firefox with specifically checking for Camino but hey, let check for the other as well ;)